### PR TITLE
Setup sentry #165160753

### PR DESCRIPTION
**Pull request by akegan**
https://www.pivotaltracker.com/story/show/165160753

*Task before merge*: Remove errors added to test sentry

### CODE REVIEW

File: app/controllers/api_controller.rb [hunk 1]
```diff
@@ -17,6 +21,8 @@ def render_error(opts = {})
     else
       message = 'Not found.'
     end
+    p 'yo'
```
Comment (chrisdolendo) [inline app/controllers/api_controller.rb]: let's remove this

File: app/javascript/components/listings/ListingsPage.js [hunk 2]
```diff
@@ -51,6 +52,7 @@ const layout = {
 }
 
 const ListingsPage = ({ page, listings }) => {
+  Sentry.captureException(new Error('This is my fake error message with sourcemaps hopefully'))
```
Comment (chrisdolendo) [inline app/javascript/components/listings/ListingsPage.js]: Do we want to keep this message on there or update?

File: app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -27,4 +28,5 @@ html
     == render 'layouts/icons'
 
     javascript:
-      var SALESFORCE_BASE_URL = "#{{current_user&.admin ? ENV['SALESFORCE_INSTANCE_URL'] : ENV['COMMUNITY_LOGIN_URL']}}"
\ No newline at end of file
+      var SALESFORCE_BASE_URL = "#{{current_user&.admin ? ENV['SALESFORCE_INSTANCE_URL'] : ENV['COMMUNITY_LOGIN_URL']}}"
```
Comment (chrisdolendo) [inline app/views/layouts/application.html.slim]: I think we need to remove this JS because we brought it in differently

**Review by chrisdolendo — CHANGES_REQUESTED** [review]
Just some cleanup of the code.

Also getting failed semaphore tests with Bundler.. `You must use Bundler 2 or greater with this lockfile.`

File: app/javascript/components/listings/ListingsPage.js (hunk 2 above)
Comment (akegan) [reply app/javascript/components/listings/ListingsPage.js]: This is part of the task before merge. Once you approve, I'll remove these errors before merging

**Comment by akegan** [comment]
@chrisdolendo I've updated semaphore job to work with bundler v2.0.0, looks like it works OK with master too.

I've also made other suggested updates! Semaphore will still until I remove errors that need to be removed before merge

**Review by chrisdolendo — APPROVED** [review]

**Pull request merged**